packages/bun-lambda/README.md

```diff
@@ -8,7 +8,7 @@ First, you will need to deploy the layer to your AWS account. Clone this reposit
 
 ```sh
 git clone git@github.com:oven-sh/bun.git
-cd packages/bun-lambda
+cd bun/packages/bun-lambda
```

Comment:
why?

Reply:
because `git clone` will create a `bun` folder and that way you could just copy and paste the whole block to your terminal.

Reply:
i didn'ŧ mention that, that's true